views/courses/edit-course.hbs

```diff
@@ -21,7 +21,7 @@
 		<section class="section-course">
 			{{#if course.isArchived}}
 				<div class="empty-state">
-					<img src="/images/empty-states/books_empty.svg" alt= "{{$t "courses._course.edit.img_alt.courseIsArchieved"}}" class="image" />
+					<img src="/images/empty-states/books_empty.svg" alt= "{{$t "courses._course.edit.img_alt.courseIsArchieved"}}" class="image course-empty-content-image" />
```

Comment:
<img width="1157" alt="Screen Shot 2020-08-03 at 13 00 24" src="https://user-images.githubusercontent.com/7567881/89176193-65994200-d589-11ea-9dfa-25ad905027cf.png">

I have found one issue for archived courses.
How to find this dialogue? 
Create a course, which ends in the past. the course is then automatically moved to Archived. Then go to edit the course (in order to delete e.g.). Then this broken view appears